specs/sb/api.md

```diff
@@ -573,9 +573,9 @@ existing system.
 </thead>
 <tbody>
 <tr>
-  <td>organization_guid*</td>
+  <td>context_guid*</td>
```

Comment:
This may need to be a JSON object or array, as some brokers may need both the org and space. This enables billing reconciliation when a broker sends the CF operator a bill, even when the org and/or space have since been deleted (these objects are not soft-deleted) in CF.